src/main/java/seedu/clinic/logic/parser/AddCommandParser.java

```diff
@@ -85,13 +85,13 @@ public AddCommand parse(String args) throws ParseException {
         }
     }
 
-
+    //TODO: Consideration: shall we put all these same method under the parent Command class?
+    //      Will it violate the Liskov substitution principle?
```

Comment:
Just to clarify, what are the other same methods that you are looking at to shift it under Command class? :)

Reply:
Currently it's just the `arePrefixesPresent` method, I found quite a few overlap among the command parser classes. Guessing it is a common utility method.

Reply:
Hmm, I am not sure if shifting to the parent command class would be okay, since AddCommandParser only implements a Parser interface and there is no parent class for the parser classes. Though I can also understand where you are coming from as well, I am thinking would it be out of place if we shift it to under ParserUtil class? Maybe we can discuss with others as well and see how do we want to implement it such that there would not be a case of overlapping.